scripts/17--html--forms/README.md

```diff
@@ -73,15 +98,16 @@
     </div>
     ```
 
-- Misc. inputs
-    - file: for selecting files
-    - search: rounded corners/clear field button
-    - color
+- Miscellaneous types
+    - `file`: for selecting files
+    - `search`: rounded corners/clear field button
+    - `color`
+    - `hidden`: data that needs submitting but not show to user
```

Comment:
I feel like `hidden` maybe deserves its own section? Maybe not. But they're used a lot in back-end code for things like nonces/tokens.